### PR TITLE
Records of Eminence Season Pass Expansion Pack DLC #1

### PR DESCRIPTION
**Pull request by Apocarypse**
For the low cost of my sanity and carpal tunnel, I bring you great news!
Previously, only a few categories had records implemented, but I have greatly increased that number.

The following categories have been fully implemented:

Spoils 1
Spoils 2
Combat (Original Areas 1)
Combat (Original Areas 2)
Combat (Original Areas 3)
Combat (Original Areas 4)
Combat (Zilart 1)
Combat (Zilart 2)
Combat (Promathia 1)
Combat (Promathia 2)
Combat (Aht Urhgan)
Combat (Goddess 1)
Combat (Goddess 2)
Combat (Abyssea)

This adds around 196 records to the game, bringing our running total to 256.

Enjoy!

**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

**Comment by zach2good** [comment]
Some build errors snuck into `feature/roe` in the last round, which I ninja-patched a few days ago. Pull `feature/roe` into your branch to get working CI 👍

**Review by zach2good — APPROVED** [review]
Content and formatting, all seems in order.

I'm starting to smell the need to split these out into separate files 👃

**Comment by zach2good** [comment]
> Files changed: 1

(if you get another person to sign off on this and update your build I could merge this and ship it into canary today because it's so self-contained)

**Comment by Kreidos** [comment]
Yup all looks good. :+1: 

DLC number 2 will be adding more, and I will be parting the records out into their own file for sure. :laughing:

**Review by Kreidos — APPROVED** [review]

**Comment by Kreidos** [comment]
Also @zach2good it looks like this is based on the tip of feature/roe, so not sure what's up with the CI? :shrug:

**Comment by zach2good** [comment]
_yeet_

File: scripts/globals/roe.lua [hunk 1]
```diff
@@ -437,7 +437,473 @@ tpz.roe.records =
...
+        trigger = triggers.mobKill,
+        reqs = { mob = set{17195318} },
+        reward = { sparks = 500, xp = 1000 },
+    },
+
+
+    [237 ] = { -- Conflict: Ordelle's Caves
+        trigger = triggers.mobKill,
+        goal = 10,
+        reqs = { zone = set{102} },
```
Comment (ibm2431) [inline scripts/globals/roe.lua:897]: Quickly browsing PRs a month later: Zone ID got c/ped from La Theine above~

File: scripts/globals/roe.lua [hunk 2]
```diff
@@ -605,6 +1071,943 @@ tpz.roe.records =
...
+    [287 ] = { -- Subjugation: Serket
+        trigger = triggers.mobKill,
+        reqs = { mobID = set{17596720} },
+        reward = { sparks = 500, xp = 1000 },
+    },
+
+    [288 ] = { -- Conflict: Castle Oztroja
+        trigger = triggers.mobKill,
+        goal = 10,
+        reqs = { zone = set{200} },
```
Comment (ibm2431) [inline scripts/globals/roe.lua:1237]: 200 is Garlaige from above

**Pull request merged**